### ISSUE TITLE
Add view support to allow different viewing angles (potentially also color scheme or  grid lines)

### ISSUE THREAD

**Issue by donnels** [pschatzmann/jupyter-openscad-kernel#7]
I may be missing something but I couldn't find a way to easily change the view presented of the objects.
I know that as the command line is being used it should be possible either through command line operations or through use of the $vpd etc variables. I tried to use the variables to change distance or camera angle but they didn't take effect from inside the rendered code.
So now I wonder is it possible to easily pass command line options to openscad through the notebook so as to achieve that effect. 

p.s. I like the openscad jupyter kernel. Thank you for doing it.

**Comment by pschatzmann** [comment]
I think this is related to https://github.com/openscad/openscad/issues/1797

I tested it with OpenSCAD version 2021.01 and can confirm that the following is working perfectly now:

$vpt = [ 0,0,0 ];
$vpr = [ 40,0,20 ];
$vpd = 200;

cube([20,20,20]);
%display

**Comment by donnels** [comment]
Thanks for the heads up I've been using the standard openscad with debian. so that is probably the problem I have. it is a newer version but I am not sure it has that fix. It probably makes sense for me to build openscad from source as opposed to using the default installed with apt. As I had set up my own docker file to make things fly on my ARM pinebook I"ll have a look if that works and then potentially give you some feedback. I had already started to set it up as multi stage so that fits well as I can build openscad in an early stage and then pass the binary on. I can also pass on the dockerfile when it works.
